lib/modules/apostrophe-ui/public/js/ui.js

```diff
@@ -25,6 +25,59 @@ apos.define('apostrophe-ui', {
 
     self.options = options;
 
+    // Utility that lets you add a tooltip to any HTML element using
+    // the data-apos-tooltip="YR TOOLTIP MSG" pattern
+
```

Comment:
After reading this I am still not sure what the difference between "prep" and "complete" is.